IdentityCore/src/webview/systemWebview/ios/MSIDSystemWebviewController.h

```diff
@@ -41,6 +41,15 @@
        allowSafariViewController:(BOOL)allowSafariViewController
                          context:(id<MSIDRequestContext>)context;
 
+- (instancetype)initWithStartURL:(NSURL *)startURL
```

Comment:
nit: I really like how you're following Apple's sample in declaring some APIs iOS 13 only. However, those are internal APIs and introducing new ones for each iOS release (if they add new params) will make the code a bit difficult to read long term. Therefore, I'm totally fine in having one common method that accepts all parameters including new ones for all versions if it simplifies.

Reply:
done